fuel-core-interfaces/src/db.rs

```diff
@@ -66,6 +66,7 @@ impl From<KvStoreError> for InterpreterError {
 pub mod helpers {
 
     use async_trait::async_trait;
+    use chrono::Utc;
```

Comment:
Any reason to favor `chrono` over `time`? `time` is more maintained with better interface compared to chrono, that relies in an old version of `time` (0.1)

Reply:
I'm in favor of finding an alternative to chrono, but this can be deferred to a tech debt item.

Reply:
created issue: https://github.com/FuelLabs/fuel-core/issues/384
I would use `u64` as it is easier to serialize as a native type.